### ISSUE TITLE
this test dai faucet is not longer on https://testnet.aave.com/faucet. where is it now?

### ISSUE THREAD

**Issue by willvlad** [aave/aave-protocol#57]
this test dai faucet is not longer on https://testnet.aave.com/faucet. where is it now?

**Comment by kyzia551** [comment]
Hey @willvlad!

If you're interested in the v1 DAI faucet, you can find it here: testnet-v1.aave.com/faucet, but your wallet should be connected to the kovan network.

If you need it for v2, then https://testnet.aave.com/faucet is the right URL, but select v2 (k) - kovan market first

**Comment by willvlad** [comment]
Hey @kyzia551 thanks

when you say "if you need it for v2, then https://testnet.aave.com/faucet is the right URL, but select v2 (k) - kovan market first" what exactly do you mean? As you can see below, I already have V2 selected and kovan connected, but I am not seeing any faucets in the UI at all:

<img width="1901" alt="Screenshot 2021-06-10 at 11 28 14" src="https://user-images.githubusercontent.com/8473897/121509917-05600180-c9df-11eb-982f-fe983ca710ed.png">

**Comment by kyzia551** [comment]
Yes, but you have mainnet market selected, but you should select the left one with k
![image](https://user-images.githubusercontent.com/17529306/121516923-701d3700-c9f7-11eb-80b3-c4f29cf4fa5e.png)

I think we will do some minor updates quite soon, and remove mainnet markets from these deployments to not confuse people.
